roles/openshift-patch-resource/README.md

```diff
@@ -0,0 +1,41 @@
+openshift-patch-resource
+========================
+
+Patch a Kubernetes/Openshift resource
+
+Requirements
+------------
+
+This role requires the `kubernetes` python module
+
+Role Variables
+--------------
+
+| Variable | Description | Required | Defaults |
+| :------- | :---------- | :------- | :------- |
+| api_version | The api_version of the resource | yes ||
+| definition | The patch definition | yes ||
+| kind | The kind of the resource | yes ||
+| merge_type | A list of merge types ("strategic-merge, merge") | no | strategic-merge |
+| name | The name of the resource | yes ||
+| namespace | The namespace of the resource | no ||
+
+Example Playbook
```

Comment:
Could we include some example vars here as well? ... or maybe just point to the tests to see some examples?